README.md

```diff
@@ -1,104 +1,130 @@
+const MMprovider = await detectEthereumProvider();
+const mumbai = providers.providers.getNetwork(80001);
 
-See `./examples/api-example.ts` for an abridged API example.
+if (MMprovider) {
+  const web3Provider = new providers.providers.Web3Provider(
+    MMprovider,
+    mumbai
+  );
+  const newDeployed = await newStrategy.deploy('test', web3Provider);
+} 
+```
 
-See [`nucypher-ts-demo`](https://github.com/nucypher/nucypher-ts-demo) for usage example with React app.
+> :warning: Deploying a Strategy requires writing to the blockchain. This requires a funded wallet and connection to the blockchain via a `provider`(e.g. MetaMask).
 
-### Using Threshold Decryption
+## 6. Encrypt the plaintext
 
-There are several schemes available for the user.
-They follow a naming convention of `<NETWORK>-<M>-of-<N>`, where `M` is the threshold required, and `N` is the total size of the cohort.
-`NETWORK` is either `mainnet` or `ibex`.
+We're now able to encrypt data to this newly deployed Strategy – which implies future access to this data will be based on ownership of the previously specified NFT, and nothing else. We'll now encrypt a plaintext using the encryptor object:
 
-An encrypter can be created by:
-```js
-import { makeTDecEncrypter } from '@nucypher/nucypher-ts'
+```javascript
+const encrypter = newDeployed.encrypter;
 
-const encrypter = await makeTDecEncrypter("ibex-2-of-3");
+const plaintext = 'this is a secret';
+const encryptedMessageKit = encrypter.encryptMessage(plaintext);
 ```
 
-The equivalent decrypter can be created by:
-```js
-import { makeTDecDecrypter } from '@nucypher/nucypher-ts'
+## 7. Request decryption rights
+
+Finally, we'll test the access control service by submitting a request to the network:
 
-// you can specify your own porter url here
-const decrypter = await makeTDecDecrypter("ibex-2-of-3", "https://porter-ibex.nucypher.community")
+```javascript
+const decrypter = newDeployed.decrypter;
+
+const decryptedMessage = await decrypter.retrieveAndDecrypt([
+  encryptedMessageKit,
+]);
 ```
 
-Please note, the schemes for the encrypter and decrypter **must** match.
+At decryption time, the requester will be asked to verify their address by signing a message in MetaMask. If they own the correct NFT, the message will decrypt successfully.
 
-Encryption and decryption is then as simple as:
-```js
-const plaintext = toBytes('plaintext-message');
-const encryptedMessageKit = encrypter.encryptMessage(plaintext);
+For more guidance on Cohort, Condition and Strategy object reuse and customization, check out the [References](https://docs.threshold.network/app-development/threshold-access-control-tac/references) page.
+
+# Contributing
 
-const decryptedPlaintext = await decrypter.retrieveAndDecrypt([encryptedMessageKit]);
-```
\ No newline at end of file
+If you would like to get involved with the development of `nucypher-ts`, please see our [Contributing Guide](https://docs.threshold.network/app-development/contribution-guide).
```

Comment:
We should put this "Contribution Guide" as a sub-page under the `Threshold Access Control` section in gitbook, and not its own section.

Reply:
On second thought, it's unclear to me that the Contribution Guide in the threshold docs is correct. Currentlym it only provides information about `nucypher-ts`, but TAC involves:
- `nucypher`
- `nucypher-ts`
- `nucypher-core`
- `nucypher-porter`

Unless that contributing page in the threshold docs is going to be updated, perhaps just add a [contributing guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) to the `nucypher-ts` repos, and link to it.

It is good to have contributing guides in the repos, since this where devs live. The Threshold Docs can perhaps provide a general overview of stuff for TAC and link out to the contribution guides in the various repos...?

(cc @arjunhassard )

Reply:
sure, i like that